### ISSUE TITLE
Any performance data

### ISSUE THREAD

**Issue by xziya** [lu-ym/cuda_test#1]
This is a good project and startup. I am wondering if there is any performance data of your program?

**Comment by yingluAMD** [comment]
No. This is only a simple tour in CUDA. Expecting next projects...
